sys-apps/eza/eza-0.18.21.ebuild

```diff
@@ -209,7 +209,10 @@ IUSE="+git"
 
 DEPEND="git? ( >=dev-libs/libgit2-1.8.1:= )"
 RDEPEND="${DEPEND}"
-BDEPEND=">=virtual/rust-1.70.0"
+BDEPEND="
+	>=virtual/rust-1.70.0
+	<virtual/rust-1.80.0
+"
```

Comment:
IMO there is no need for this third commit.

Reply:
0.18.21 fails to build with rust 1.80. As explained my bug report (937921), this is an upstream issue that was resolved in 0.18.24.

Why do you think it's unnecessary? (Sorry if this sounds rude, I'm not trying to be.)

Reply:
Requiring upper bounds on a build dependency is a bit of an ugly situation IMHO. In this case I think it's unlikely people will accept keywords for rust but not eza, and the real solution here is to simply upgrade eza.

For the older eza version we can rely on the likelihood that people will run a stable system when trying to build both packages. For the newer version I think we should just drop it as well, since:
- it has no stable keywords
- anyone using it with unstable keywords can simply use the very latest version
- there's little point trying to stabilize a version with a known serious issue

Reply:
That makes sense. There's no point in keeping 0.18.21 around since users running ~arch are just going to upgrade to the next version when it's available and users running stable are running an older rust so they are unaffected.